### PR TITLE
Add reroll functionality to standard rolls.

### PR DESCRIPTION
**Pull request by ashein**
This should add the "r" or "r + number" suffix notation to standard dice rolls so that they would reroll silently if at or below the given threshold (defaults to one). This modifies standard roll routines, but doesn't seem to break anything so far.
UP: now also adds `(reroll)` or `(N rerolls)` after the roll results in case one or more rerolls were triggered.

### CODE REVIEW

File: Dicebot/plugin.py [hunk 1]
```diff
@@ -121,7 +153,7 @@ def _process(self, irc, text):
                 (self.rollReDH, self._parseDHRoll),
                 (self.rollReWG, self._parseWGRoll),
                 ]
-        results = []
+        results = [ ]
```
Comment (wRAR) [inline Dicebot/plugin.py]: лишнее

File: Dicebot/plugin.py [hunk 2]
```diff
@@ -141,48 +173,86 @@ def _parseStandardRoll(self, m):
...
         for _ in range(rolls):
             result = totalMod
-            for sides, dice in totalDice.items():
-                if sides > 0:
-                    result += self._roll(dice, sides)
+            for key, dice in totalDice.items():
+                # Unpack the key
+                opts = dieSpec.match(key)
+                sides = int(opts.group('sides'))
+                reroll = opts.group('reroll') and int(opts.group('reroll')) or 0
```
Comment (wRAR) [inline Dicebot/plugin.py]: `int(opts.group('reroll') or 0)`

File: Dicebot/plugin.py [hunk 3]
```diff
@@ -312,6 +382,7 @@ def _parse7SeaRoll(self, m):
         return '[%dk%d%s%s] %s' % (rolls, keep, self._formatMod(mod), explodeStr,
                                    '; '.join(results))
 
+
```
Comment (wRAR) [inline Dicebot/plugin.py]: лишнее

**Review by wRAR — CHANGES_REQUESTED** [review]

**Comment by ashein** [comment]
Not sure how to mark the lines where changes were needed as "resolved".

File: .project [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wRAR) [inline .project]: весь этот коммит не нужен

**Review by wRAR — CHANGES_REQUESTED** [review]

**Pull request closed without merging**